zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/ResponseMapper.java

```diff
@@ -233,6 +235,13 @@ private static ResponseEntity<Object> buildCreateProcessInstanceResponse(
     return new ResponseEntity<>(response, HttpStatus.OK);
   }
 
+  public static ResponseEntity<Object> toSignalBroadcastResponse(
+      final SignalRecord brokerResponse) {
+    final var response =
+        new SignalBroadcastResponse().key(1L).tenantId(brokerResponse.getTenantId());
```

Comment:
❌ I'm waiting for https://github.com/camunda/camunda/pull/21971 to be merged in order to return the proper key